### ISSUE TITLE
Remove replaces from go.mod

### ISSUE THREAD

**Issue by migmartri** [vmware-tanzu/asset-relocation-tool-for-kubernetes#58]
We have some module replaces in our main go.mod, we need to figure out a way to move away from them and rely on referencing upstream libraries. The most difficult one will be to move to upstream yaml.v3

```bash
$ go install github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes@v0.1.0
go install github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes@v0.1.0: github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes@v0.1.0
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

**Comment by josvazg** [comment]
That one in particular depends on us not using directly `github.com/atomatt/yaml` right?
Didn't we have an issue for that already?

**Comment by petewall** [comment]
This was closed because of PR #68 , but that only solves 2 out of the 3 replace rules. Reopening...

**Comment by petewall** [comment]
Closing as the remaining of this issue will be solved by #91
